test/unit/dialects/sqlite/data-types.test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const chai = require('chai'),
+  expect = chai.expect,
+  Support = require('../../support'),
+  dialect = Support.getTestDialect(),
+  BaseTypes = require('sequelize/lib/data-types'),
+  DataTypes = require('sequelize/lib/dialects/sqlite/data-types')(BaseTypes);
+
+if (dialect.match(/^sqlite/)) {
+  describe('[SQLITE Specific] DataTypes', () => {
```

Comment:
Are these tests really sqlite specific? Is the behaviour different on other dialects?

Reply:
Ok,Here's what I think。

1. I don't know the details of how DATE is handled in other dialects, but it appears from the source code that different dialects handle dates differently from each other.
2. the problem I'm having only occurs in sqlite, so I'm just fixing the parse function in sqlite and making sure it's working.

Reply:
I think the other dialects should also handle these test cases the same way, so I'd like to see these tests running on all dialects and therefore be added to an existing test suite